public/js/readMore.js

```diff
@@ -0,0 +1,18 @@
+var acc = document.getElementsByClassName("readMoreToggle");
+var i;
+
+for (i = 0; i < acc.length; i++) {
```

Comment:
Could clean this up a bit by doing a for of instead.
`
for (const toggleEl of document.getElementsByClassName("readMoreToggle")) {
   toggleEl.addEventListener(....
}
`